doc/releases/changelog-0.25.0.md

```diff
@@ -156,16 +157,35 @@
   >>> y = np.array([np.pi/6, np.pi/5], requires_grad=True)
   >>> qml.gradients.param_shift(circuit, broadcast=True)(x, y)
   (tensor([[-0.7795085,  0.       ],
-          [ 0.       , -0.7795085]], requires_grad=True),
-  tensor([[-0.125,  0.   ],
-          [ 0.   , -0.125]], requires_grad=True))
+           [ 0.       , -0.7795085]], requires_grad=True),
+  tensor([[-0.125, 0.  ],
+          [0.  , -0.125]], requires_grad=True))
   ```
 
+  The following simpler example also makes use of broadcasting:
+
+  ```python
+  @qml.qnode(dev, diff_method="parameter-shift", broadcast=True)
+  def circuit(x, y):
+      qml.RX(x, wires=0)
+      qml.RY(y, wires=1)
+      return qml.expval(qml.PauliZ(0) @ qml.PauliZ(1))
+  ```
+
+  ```pycon
+  >>> x = np.array(0.1, requires_grad=True)
+  >>> y = np.array(0.4, requires_grad=True)
+  >>> qml.grad(circuit)(x, y)
+  (array(-0.09195267), array(-0.38747287))
+  ```
+
+  where only 2, instead of 4 with `broadcast=False`, circuits are created internally.
+
   To illustrate the speedup, for a constant-depth circuit with Pauli rotations and controlled Pauli rotations, the 
   time required to compute `qml.gradients.param_shift(circuit, broadcast=False)(params)`
   ("No broadcasting") and `qml.gradients.param_shift(circuit, broadcast=True)(params)` 
   ("Broadcasting") as a function of the number of qubits is given
-  [here](https://pennylane.readthedocs.io/en/stable/_images/default_qubit_native_broadcast_speedup.png).
+  [here](https://pennylane.readthedocs.io/en/stable/_images/param_shift_broadcast_speedup.png).
```

Comment:
I assume this will work when v0.25 is the "stable" version? 🤞

Reply:
I think you are right 👍